ConsoleView.cpp

```diff
@@ -1,26 +1,41 @@
+//All these functions return void, they are simply here 
+//for the user to call when they want to get information on 
+//a CLASS (or a list of all classes), RELATIONSHIP, or ATTRIBUTE
 
-int UserInputLoop()
-{
-    std::vector<UMLClass> Classes;
-    int count = 0;
-    std::cout << "Enter the amount of Classes to add:" << std::endl;
-    std::cin >> count;
+//Prtint class will be used to print the name of a class given a UMLClass object
+//By using the get_class_name function from the UMLClass, it then prints the name of the Class given
+//@Para: UMLClass OBJ
+PrintClassName(UMLClass X){
+    
+    std::cout<< x.get_class_name(); 
```

Comment:
May want to add an std::endl here, but if you think this is right then thats cool too.